Library/Formula/ratfor.rb

```diff
@@ -0,0 +1,15 @@
+class Ratfor < Formula
+  version "1.02"
+  desc "Rational Fortran"
+  homepage "http://www.dgate.org/ratfor/"
+  url "http://www.dgate.org/ratfor/ratfor.tar.gz"
```

Comment:
We don't really use static URLs in the core. It needs to be a versioned link that won't silently break on us.